llama_stack/providers/registry/vector_io.py

```diff
@@ -54,6 +54,14 @@ def available_providers() -> List[ProviderSpec]:
             config_class="llama_stack.providers.inline.vector_io.faiss.FaissImplConfig",
             api_dependencies=[Api.inference],
         ),
+        InlineProviderSpec(
+            api=Api.vector_io,
+            provider_type="inline::sqlite_vec",
+            pip_packages=EMBEDDING_DEPS + ["sqlite_vec"],
```

Comment:
```suggestion
            pip_packages=EMBEDDING_DEPS + ["sqlite-vec"],
```

Even if pip converts the `_` to `-ù I believe that's clearer?

Reply:
Sounds good!